src/main/java/com/p6spy/engine/wrapper/ConnectionWrapper.java

```diff
@@ -55,27 +55,30 @@
 public class ConnectionWrapper extends AbstractWrapper implements Connection {
 
   private final Connection delegate;
-  private final JdbcEventListener eventListener;
+  private final JdbcEventListener jdbcEventListener;
   private final ConnectionInformation connectionInformation;
 
-  public static Connection wrap(Connection delegate, JdbcEventListener eventListener, ConnectionInformation connectionInformation) {
```

Comment:
I know I'm too late for this, but we should have probably deprecated the `public static Connection wrap` method instead of removing it as this is a breaking change. The reason the constructor was private is that the connection might be null. in this case, wrap also returns null to avoid NPEs. This is now moved to the `public Connection wrap()` but calling this method is not enforced.